controllers/cohortdefinition.go

```diff
@@ -55,6 +56,25 @@ func (u CohortDefinitionController) RetriveAll(c *gin.Context) {
 		c.Abort()
 		return
 	}
-	c.JSON(http.StatusOK, gin.H{"CohortDefinition": cohortDefinitions})
+	c.JSON(http.StatusOK, gin.H{"cohort_definitions": cohortDefinitions})
 	return
 }
+
+func (u CohortDefinitionController) RetriveStatsBySourceId(c *gin.Context) {
```

Comment:
Is there logic elsewhere that reduces this to only the cohorts you are authorized to access?

Reply:
Good question. I did not add anything specific in this PR. All authorization seems to be limited to the section you pointed out in our chat: https://github.com/uc-cdis/cohort-middleware/blob/omop_endpoints_and_improved_docs/middlewares/auth.go#L29

Reply:
for the record: we've decided to follow-up on this in a separate ticket+PR.